src/goto-symex/symex_main.cpp

```diff
@@ -57,7 +57,7 @@ symex_configt::symex_configt(const optionst &options)
             : DEFAULT_MAX_FIELD_SENSITIVITY_ARRAY_SIZE),
     complexity_limits_active(
       options.get_signed_int_option("symex-complexity-limit") > 0),
-    cache_dereferences{options.get_bool_option("symex-cache-dereferences")}
+    cache_dereferences{true /* options.get_bool_option("symex-cache-dereferences") */}
```

Comment:
:no_entry_sign: This change appears unrelated to the commit message. Was it included by accident?